### DISCUSSION TITLE
Add directly to yaml

### DISCUSSION THREAD

**Discussion by imed-ghomari** [mdelobelle/metadatamenu#61] [Ideas]
Hi, I'm always using using yaml for metadata, and everytime I add a new property, I have to select WHERE to put it.

I don't care where the property is created as long as it's in the yaml.

Could we have an option in the settings to always append new properties to yaml ? (without selecting its location)

**Comment by mdelobelle** [comment]
Agreed this is a good idea. I'll create an issue to put this on the roadmap

**Comment by mdelobelle** [comment]
this is now available in version 0.1.16

Doc here: https://github.com/mdelobelle/metadatamenu#infrontmatter

**Reply by imed-ghomari** [reply]
Thank you, I'm away now but when I get back I'll try this new option.

**Comment by javiavid** [comment]
I'm interested in this option, but I can't find it.

What I want is that the selection window to choose where to save the field does not appear, I always use YAML.
Is this possible?
